packages/ui-components/src/components/toaster/toaster.scss

```diff
@@ -21,10 +22,11 @@
 	 * @prop --toaster-icons-width: toaster icons width
 	 */
 
-	--toaster-large-height: 74px;
-	--toaster-small-height: 58px;
+	--toaster-large-height: 72px;
+	--toaster-small-height: 56px;
 
-	--toaster-width: 624px;
+	--toaster-width: 500px;
+	--toaster-top-space: 16px;
```

Comment:
should this action propagate over all application?

Reply:
yes